.github/workflows/containers.yaml

```diff
@@ -42,22 +40,19 @@ jobs:
           set -x
           TAGS=""
           if [[ '${{ github.event_name }}' == 'push' ]]; then
-            TAGS="${TAGS} ${{ github.sha }}"
-            if [[ '${{ github.ref_name }}' == 'main' ]]; then
-              TAGS="${TAGS} latest"
-            else
-              TAGS="${TAGS} ${{ github.ref_name }}"
-            fi
+            TAGS="${TAGS} ${{ github.sha }} ${{ github.ref_name }}"
           elif [[ '${{ github.event_name }}' == 'pull_request' ]]; then
             TAGS="${TAGS} ${{ github.sha }}"
           elif [[ '${{ github.event_name }}' == 'schedule' ]]; then
-            TAGS="${TAGS} latest"
+            TAGS="${TAGS} latest ${{ github.repository }} --json defaultBranchRef -q '.defaultBranchRef.name')"
```

Comment:
What is this for? Doesn't mean we always push to main for scheduled runs?

Also. Where is the opening parenthesis?

Reply:
Correct - schedule would always go to the default branch

Reply:
Can't this just use the same logic? github.ref should be set too

Reply:
From my understanding `github.ref_name ` is blank when the event is `schedule`